agglayer/types.go

```diff
@@ -131,19 +153,54 @@ type SignedCertificate struct {
 	Signature *Signature `json:"signature"`
 }
 
+func (s *SignedCertificate) String() string {
+	return fmt.Sprintf("Certificate:%s, \nSignature: %s", s.Certificate.String(), s.Signature.String())
```

Comment:
Nitpick, in order to align the "labels":

```suggestion
	return fmt.Sprintf("Certificate:%s,\nSignature: %s", s.Certificate.String(), s.Signature.String())
```

Reply:
Fix: https://github.com/0xPolygon/cdk/pull/165/commits/db602e6fdd89ecedb9aca8526e2b32ef12624a57